cmd/serve.go

```diff
@@ -82,8 +82,8 @@ func init() {
 	serveCmd.Flags().Bool("lookup-enabled", false, "Use the lookup client to attempt to fetch metadata or userdata from an upstream source when it is not cached locall for the instance")
 	viperBindFlag("lookup.enabled", serveCmd.Flags().Lookup("lookup-enabled"))
 
-	serveCmd.Flags().String("lookup-base-url", "", "A base url (like 'https://metadata-lookup-service.tld/api/v1/') to use when fetching metadata or userdata from an upstream source")
-	viperBindFlag("lookup.baseurl", serveCmd.Flags().Lookup("lookup-base-url"))
+	serveCmd.Flags().String("lookup-service-url", "", "URL to the metadata service (like 'https://metadata-lookup-service.tld/api/v1/') to use when fetching metadata or userdata from an upstream source")
```

Comment:
maybe a nitpick but the comment here should probably specify it's for the "lookup" service -- as this is a flag on the metadata service itself :)